src/proto/mgmt/pool.proto

```diff
@@ -97,8 +101,9 @@ message PoolDrainResp {
 message PoolExtendReq {
 	string uuid = 1; // uuid of pool to add target up to
 	repeated uint32 ranks = 2; // ranks
-	uint64 scmbytes = 3; // SCM size in bytes
-	uint64 nvmebytes = 4; // NVMe size in bytes
+	repeated uint32 svc_ranks = 3; // List of pool service ranks
```

Comment:
Yeah the thought here was that the scmbytes/nvmebytes fields would eventually go away, because the goal is for the extend operation to be able to query for those values rather than having them supplied as part of the request.